web/themes/custom/votegov/templates/layout/page--node--voter-guide.html.twig

```diff
@@ -39,6 +39,8 @@
       </div>
     </div>
   </main>
+  <div role="region" aria-labelledby="voter-guide-cards">
```

Comment:
added this to address the error of this block not having a landmark

Reply:
I don't see the aria-labeledby related to any id of a header in the component.

Reply:
Can we set it to an aria-label of value `{ 'Know your voting rights' | t '}`?